dbt_integration.py

```diff
@@ -169,25 +167,48 @@ def wrapper(
 
     return wrapper
 
-def default_profiles_dir(project_dir) -> Path:
-    if "DBT_PROFILES_DIR" in os.environ:
-        return Path(os.environ["DBT_PROFILES_DIR"]).resolve()
-    return project_dir if (project_dir / "profiles.yml").exists() else Path.home() / ".dbt"
 
-def find_package_paths(project_directories, profiles_dir_override):
+def default_profiles_dir(project_dir):
+    if "DBT_PROFILES_DIR" in os.environ:
+        profiles_dir = os.path.expanduser(os.environ["DBT_PROFILES_DIR"])
+        if os.path.isabs(profiles_dir):
+            return os.path.normpath(profiles_dir)
+        return os.path.join(project_dir, profiles_dir)
+    project_profiles_file = os.path.normpath(os.path.join(project_dir, "profiles.yml"))
+    return (
+        project_dir
```

Comment:
We need to convert this to string right?

Reply:
Nope this should all be strings. os.path operations operatate on strings.